### ISSUE TITLE
Fix download_panel_info_downloaded_file_deleted

### ISSUE THREAD

**Issue by tracywalker** [mozilla/iris#1894]
This test case is leaving some sort of file picker window open on Windows.  It's not causing the test to fail. This window lingers around during the remainder of the test run and after. It should be closed as part of the test case.

**Comment by rrobotin** [comment]
Hi Tracy,

I've tried to reproduce it on Windows, but the test didn't left any file picker window open. Have you had any Windows explorer window/windows opened at the moment of the test run? That was the only case I was able to reproduce the issue you are reporting, but even so, the test has passed. 

Thanks

**Comment by tracywalker** [comment]
No,  the only thing I have open is a terminal.  I'll run the test group to see if I can figure out the problem.

**Comment by tracywalker** [comment]
Ok.  I see what is going on.  You're using close_tab() in a way it is not meant to be used.

63       # Close download folder window.
64       close_tab()

The reliable way is with click_window_control('close', 'auxiliary')  
I'll fix this in branch issue_1894 and also add an appropriate check to ensure the download folder is closed.

**Comment by tracywalker** [comment]
Fixed on Windows by using window controls.  That then it broke the test on Mac do to window controls being flaky. Filed issue #1917 for Ionut to try to improve.
